### PR TITLE
Display new navigation for content tagged to taxons

### PR DESCRIPTION
**Pull request by alecgibson**
As part of the rollout for the Parenting & Childcare taxonomy, we are
displaying new navigation elements on the Child Benefit Tax Calculator,
to provide a consistent UI for any users who are assigned to the “B”
bucket in the new Navigation A/B test.

This involves showing the taxon breadcrumbs, and the taxonomy sidebar.
On the landing page, the taxonomy sidebar replaces the existing sidebar.
On “main” pages, a taxonomy sidebar has been added, where there wasn’t
one before.

Note that naming has been kept to `EducationNavigation` and similar,
because that’s currently the naming in use by the cookie, and by
similar code in other applications. It has been kept like this to
maintain consistency and to assist future searches for code occurences.

## Old navigation (A)

landing-page-a
main-page-a

## New navigation (B)

landing-page-b
main-page-b

## Trello

https://trello.com/c/bQTK0OI2/90-changes-to-education-and-or-parenting-and-childcare-related-content-pages-as-part-of-the-new-navigation-in-calculators

### CODE REVIEW

File: app/views/child_benefit_tax/main.html+new_navigation.erb [hunk 1]
```diff
@@ -0,0 +1,138 @@
+<% content_for :title, "Child Benefit tax calculator" %>
+<% content_for :body_classes, "full-width" %>
+
+<div class="grid-row">
+  <main id="content" role="main" class="group calculator-page">
```
Comment (carvil) [inline app/views/child_benefit_tax/main.html+new_navigation.erb]: Is there a chance we could re-use more of the body of the page by moving that into a common partial? That would avoid content changes getting out of sync between versions.

Comment (alecgibson) [inline app/views/child_benefit_tax/main.html+new_navigation.erb]: (We can reassess after Template Consolidation merge [`to-elements`](https://github.com/alphagov/calculators/tree/to-elements))

Comment (alecgibson) [inline app/views/child_benefit_tax/main.html+new_navigation.erb]: @carvil done

**Comment by alecgibson** [comment]
@carvil good catch. I've fixed it, and added some screenshots to the PR description.

**Pull request merged**